docs/aks-cheatsheet.md

```diff
@@ -62,7 +62,7 @@ Get access credentials for a managed Kubernetes cluster (passing the
 resource group and the name):
```

Comment:
passing the *environment name*

Reply:
Please remove the "Select account for az:" section as it's taken care when you run get-cluster-credentials